scripts/gdprdeleteall.php

```diff
@@ -0,0 +1,42 @@
+#!/usr/local/bin/php -q
+<?php
+/**
+ * DO NOT RUN THIS SCRIPT WILLY NILLY
+ * this script will delete the personal data of any members that have been emailed regarding deletion and have not acted
+ * you should probably run the gdpremail script first then wait a week or two
+ * 
+ * run with "php gdprdeleteall.php"
+ */
+
+use \MyRadio\Database;
+use \MyRadio\MyRadioException;
+use \MyRadio\MyRadioError;
+use \MyRadio\MyRadioEmail;
+use \MyRadio\MyRadio\CoreUtils;
+use \MyRadio\ServiceAPI\MyRadio_User;
+
+ini_set('log_errors', 1);
+ini_set('error_log', '/tmp/php-mailparser-error.log');
+ini_set('display_errors', 'On');
+
+require_once '../src/Controllers/root_cli.php';
+
+$db = Database::getInstance();
+
+$time = strtotime("-1 year", time());
+$date = date("Y-m-d", $time);
+
+$db->query(
```

Comment:
Still pretty confident this should be using MyRadio set functions, rather than direct database bashing for the majority of cases.

Reply:
personally I'm against that. I think we'd be adding quite a lot of code to our classes which will make our codebase even more confusing without it having much benefit over the database queries.

Reply:
I'm not talking about adding specific functions for this usecase, but using the ones that are already there - setCollege, pulling out a list of shows a particular member has credits on and updating all of those, etc etc

Reply:
Future to do